.github/labeler.yml

```
@@ -21,7 +23,8 @@ area-identity:
 
 #area-infrastructure:
```

```suggestion
area-infrastructure:
  - global.json
  - .azure/**/*
  - .config/**/*
  - eng/**/*
```
This reuses the areas in CODEOWNERS that @aspnet/build cares about.